src/model/keychain.js

```diff
@@ -9,7 +11,12 @@ import logger from 'logger';
 // NOTE: implement access control for iOS keychain
 export async function saveString(key, value, accessControlOptions) {
   try {
-    await setInternetCredentials(key, key, value, accessControlOptions);
+    await setInternetCredentials(
+      key,
+      key,
+      value,
+      accessControlOptions && { accessControl: accessControlOptions }
```

Comment:
I think the fourth argument (options) into setInternetCredentials are supposed to be an object with properties like: 
    accessControl?: ACCESS_CONTROL;
    accessGroup?: string;
    accessible?: ACCESSIBLE;
    authenticationPrompt?: string | AuthenticationPrompt;
    authenticationType?: AUTHENTICATION_TYPE;
  etc

I think now it would look like: { accessControl: {
      accessControl: ACCESS_CONTROL.USER_PRESENCE,
      accessible: ACCESSIBLE.WHEN_UNLOCKED_THIS_DEVICE_ONLY,
    }
 }